.github/workflows/ruff.yml

```diff
@@ -0,0 +1,12 @@
+name: Ruff
+on: [ push, pull_request ]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: chartboost/ruff-action@v1
+      - name: Run Ruff linter
+        run: ruff check --fix
```

Comment:
this is not needed, as `ruff check` is run above, the --fix would be lost anyway, because the changes would not be committed back to the repo